packages/core/src/gamepad.ts

```diff
@@ -0,0 +1,36 @@
+import * as ds from "@devicescript/core"
+
+declare module "@devicescript/core" {
+    interface Gamepad {
+        /**
+         * The thumbstick position register if any
+         */
+        axes(): ds.ClientRegister<{ x: number; y: number }>
+
+        /**
+         * Button (or combo) register
+         */
+        button(value: ds.GamepadButtons): ds.ClientRegister<boolean>
+    }
+}
+
+ds.Gamepad.prototype.axes = function axes() {
+    let r = (this as any).__axes as ds.ClientRegister<{ x: number; y: number }>
```

Comment:
I think you can extend the ds.Gamepad interface (I know it's a class, but I think you can do interface) and add this field

Reply:
do i have to? :)

Reply:
nah ;)